website/src/docs/dashboard.md

```diff
@@ -75,6 +76,19 @@ By default Dashboard will be rendered as a modal, which is opened via clicking o
 
 String with a CSS selector for a button that will trigger opening Dashboard modal. Multiple buttons or links can be used, if it’s a class selector (`.uppy-choose`, for example).
 
+### `plugins: []`
+
+List of plugin IDs that should be shown in the Dashboard's top bar. For example, to show the Webcam plugin:
+
```

Comment:
Wouldn’t you be able to install StatusBar or similar the same way? So its not just for `acquire` plugins?